### PR TITLE
[WIN32K] Fix 'use after free' in NtGdiStretchDIBitsInternal

### PR DESCRIPTION
**Pull request by Doug-Lyons**
## Fix Crash of VS2010 Pro Installer

_Correct 'use after free' in dibobj.c._

JIRA issue: [CORE-17861](https://jira.reactos.org/browse/CORE-17861)

## Relocate ExFreePoolWithTag for pbmiSafe below last use

### CODE REVIEW

**Review by ThFabba — APPROVED** [review]

File: win32ss/gdi/ntgdi/dibobj.c [hunk 1]
```diff
@@ -1505,6 +1504,7 @@ NtGdiStretchDIBitsInternal(
         LinesCopied = pbmiSafe->bmiHeader.biHeight;
     }
 
+    ExFreePoolWithTag(pbmiSafe, 'imBG');
```
Comment (HBelusca) [inline win32ss/gdi/ntgdi/dibobj.c:1507]:
```suggestion
    ExFreePoolWithTag(pbmiSafe, 'imBG');

```

**Review by HBelusca — APPROVED** [review]
LGTM
(Side note, not for this PR: This function seems to leak some allocated buffers in some failure code paths.)

File: win32ss/gdi/ntgdi/dibobj.c (hunk 1 above)
Comment (Doug-Lyons) [reply win32ss/gdi/ntgdi/dibobj.c:1507]: Fixed. Thanks.

**Comment by Doug-Lyons** [comment]
Thanks, @HBelusca for merging this.

**Pull request merged**